Jenkinsfile

```diff
@@ -1,73 +1,18 @@
-@Library('socrata-pipeline-library')
+@Library('socrata-pipeline-library@6.0.0') _
 
-String lastStage
-
-def dockerize = new com.socrata.Dockerize(steps, 'image-resize-service', BUILD_NUMBER)
-
-pipeline {
-  options {
-    timeout(time: 60, unit: 'MINUTES')
-    ansiColor('xterm')
-  }
-  parameters {
-    string(name: 'AGENT', defaultValue: 'build-worker', description: 'Which build agent to use?')
-  }
-  agent {
-    label params.AGENT
-  }
-  triggers {
-    issueCommentTrigger('^retest$')
-  }
-  environment {
-    SERVICE = 'image-resize-service'
-    SERVICE_VERSION = '0.0.1'
-    WEBHOOK_ID = 'WORKFLOW_ACCESS_CONTROL_Q_AND_A'
-  }
-  stages {
-    stage('Pull Request') {
-      when { changeRequest() }
-      steps {
-        sh 'sbt assembly'
-      }
-    }
-    stage('Build') {
-      when { branch 'main' }
-      stages {
-        stage('Build and push image') {
-          steps {
-            script {
-              lastStage = env.STAGE_NAME
-              sh 'sbt assembly'
-
-              // NOTE: If you're changing the version of this service, you'll also have to change the version here
-              sh 'cp target/scala-2.12/image-resize-service-assembly-0.0.1.jar image-resize-service-assembly.jar'
-              sh 'docker build --tag image-resize-service:latest .'
-              sh 'rm image-resize-service-assembly.jar'
-
-              env.DOCKER_TAG = dockerize.push_tagged_image_to_all_repos('image-resize-service:latest', env.SERVICE_VERSION, env.GIT_COMMIT)
-              currentBuild.description = env.DOCKER_TAG
-            }
-          }
-        }
-        stage('Deploy to staging') {
-          steps {
-            script {
-              lastStage = env.STAGE_NAME
-              marathonDeploy(serviceName: env.SERVICE, tag: env.DOCKER_TAG, environment: 'staging')
-            }
-          }
-        }
-      }
-      post {
-        failure {
-          script {
-            teamsWorkflowMessage(
-              message: "[${currentBuild.fullDisplayName}](${env.BUILD_URL}) has failed in stage ${lastStage}",
-              workflowCredentialID: WEBHOOK_ID
-            )
-          }
-        }
-      }
-    }
-  }
-}
+commonPipeline(
+    jobName: 'image-resize-service',
+    language: 'scala',
+    projects: [
+        [
+            name: 'image-resize-service',
+            compiled: true,
+            deploymentEcosystem: 'marathon-mesos',
+            paths: [
+                dockerBuildContext: 'docker',
+            ],
+            type: 'service',
+        ],
+    ],
+    teamsChannelWebhookId: 'WORKFLOW_ACCESS_CONTROL_Q_AND_A',
```

Comment:
ah, old team name. any plans to update these?  Not necessary for this PR, just curious.

Reply:
This is really up to each team to change, although I am happy to help with the process.
There are some options based on what's appropriate:
- If a team renames a channel, then the same teams webhook, and therefore Jenkins credential, will continue to work.  I can also create a new one with the name that matches to reduce confusion.
- If a team wants a new channel to get notifications, then I can help set up a Jenkins credential.

Reply:
Also, there are two that might be appropriate here.  Do you know which one is preferred?
- WORKFLOW_ACCESS_CONTROL_NOTIFICATIONS (internal channel)
- WORKFLOW_ACCESS_CONTROL_Q_AND_A (external channel)

Reply:
ah, okay.  And good call out, both channels have benefits.  Will ask the team tomorrow which we prefer.

Reply:
I have updated it to WORKFLOW_ACCESS_CONTROL_NOTIFICATIONS, but let me know if you want it switched back.

Reply:
Oh drat. I forgot to write back.  I did talk to a couple folks during our onsite last week, but we couldn't figure out which we'd prefer. So, whatever is fine.